### ISSUE TITLE
Aurora g910 spectrum equalizer Wont work

### ISSUE THREAD

**Issue by MagnetiC77** [antonpup/Aurora#875]
### What are you experiencing an issue with?

- [ ] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [x] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [ ] Crash
- [ ] Performance problems
- [x] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
Explain here what should happen.
it should work with my music or any sound ! but it wont until i unistal or change my playback ( i cant change em :) )
### Actual behavior:
Explain here what happened instead.
i have z270 msi (realtek 1220) when realtek driver is not install program work just fine ! but when i install realtek driver it wont work same happen for logitech spectrogram 
### How often does this problem occur?

This problem occurs... (Fill in here)

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. unistall realtek driver it work fine 
2. 
3. 

### What is the version of Aurora you are experiencing this issue with?

My version is... (Fill in here)

### Does the previous version of Aurora have this issue?
The previous version, (Fill in the version here), did/did not have this issue.

### Please include any logs, screenshots, or files that are related to this issue:

For logs, you can use: http://pastebin.com/
For screenshots, you can use: http://imgur.com/
For files, you can use: http://www.tinyupload.com/ , https://www.dropbox.com/ , or attach them to the issue.

**Comment by simon-wh** [comment]
If it stops working when using those drivers it must be it interfering with the way Aurora achieves the effect. Are you able to keep those drivers uninstalled?

**Comment by MagnetiC77** [comment]
its better to fix this , and i need 5.1 channel for gaming ! so i need to install realtek drivers (Nahimic and Nahimic2+)
In playback when i change or disable speakers it work !
but when driver is install and enable equalizer wont work (both logitech spectrogram and Aurora)
logitech spectrogram show error 0x88890008
i have z270 pro carbon - realtek 1220

**Comment by MagnetiC77** [comment]
i send an email to MSI tech support ! i thing their app wont let our keyboards read the sound ! most of people have this with realtek installed ! pls help me to fix this problem :)

**Comment by cubeslice** [comment]
I discovered that Aurora's visualizer only works with the default Windows audio driver. That is why my monitor's speakers work and my headphones don't work.

**Comment by MagnetiC77** [comment]
i know ! but they should fix this !

**Comment by zejjnt** [comment]
Yeah I just plugged in my Cambridge Audio DACMagic XS and there's not a single thing happening to the backlight.
Is it too much to ask to get the audiophile experience while having the keyboard light up like I'm 14 years old? :(

**Comment by angelaly11** [comment]
i had issues with aurora's equalizer back in dev2, and it was super frustrating tbh. anyway, @MagnetiC77, i think i know what might be going on. from what you said about logitech spectrogram showing error 0x88890008 when realtek drivers are installed, idk if that's the root cause or not. but i did find a thread on this exact issue on https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 - basically, people were having issues with aurora and logitech's spectrogram app when realtek drivers were installed. it turns out the realtek driver was causing conflicts with aurora. 

one of the users on that thread suggested uninstalling the nahimic software (if you have it installed) or disabling it in device manager, then restarting your pc. also, try checking if there are any updates for the realtek drivers and make sure they're properly installed.

hope this helps!
